scripts/process_vulnerabilities.py

```
@@ -343,6 +430,15 @@ def main():
         print("No vulnerabilities meet the severity threshold")
         return
 
+    # Apply max_issues limit
+    vulns_to_process = filtered_vulns[:max_issues]
+    remaining_vulns = filtered_vulns[max_issues:]
+
+    if remaining_vulns:
+        print(f"⚠️  Limiting issue creation to {max_issues} vulnerabilities (out of {len(filtered_vulns)} total)")
+        print(f"   Remaining {len(remaining_vulns)} vulnerabilities will be logged but not converted to issues")
+        print(f"   Increase 'max_issues' input to create more issues")
```

Ruff (0.14.13)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)